wgsl/index.bs

```diff
@@ -6668,6 +6668,128 @@ When an identifier is used as a [=syntax/callable=] item, it is one of:
     | [=syntax/bitwise_expression=]
 </div>
 
+## Operator Precedence and Associativity ## {#operator-precedence-associativity}
+
+This entire subsection is non-normative.
+
+Operator precedence and associativity in [=syntax/expression|right-hand side WGSL expressions=]
+emerge from their grammar in summary. Right-hand expressions group operators to organize them,
+as illustrated by the following diagram:
+
+<div class=operators1>
+  <figure>
+    <figcaption>Operator precedence and associativity graph</figcaption>
+    <object type="image/svg+xml" data="img/operators_1.mmd.svg"></object>
```

Comment:
Now that the diagram is wide the text has gotten quite small. Ideally it wouldn't be too far off from the page body text size. Can we improve this somehow?

Reply:
Thanks for pointing out, I'll look into it! Indeed, it is important for these to match with text size.